client/src/components/DataFiles/DataFiles.jsx

```diff
@@ -114,10 +114,12 @@ const DataFiles = () => {
   const loading = useSelector((state) => state.systems.storage.loading);
   const error = useSelector((state) => state.systems.storage.error);
   const systems = useSelector((state) => state.systems.storage.configuration);
-
+  const noPHI = useSelector(state => state.workbench.config.noPHI);
```

Comment:
Perhaps we could rename this to convey it is expecting a systemId
```suggestion
  const noPHISystem = useSelector(state => state.workbench.config.noPHISystem);
```